### PR TITLE
[new release] lintcstubs (2 packages) (0.4.7)

### PR DESCRIPTION
**Pull request by edwintorok**
OCaml C stub static analyzer

- Project page: <a href="https://github.com/edwintorok/lintcstubs">https://github.com/edwintorok/lintcstubs</a>

##### CHANGES:

* lower bounds and 32-bit build fixes

### CODE REVIEW

**Comment by edwintorok** [comment]
I've also submitted a PR to upstream goblint to fix the lower bounds there: https://github.com/goblint/analyzer/pull/1157

File: packages/goblint/goblint.2.1.0/opam [hunk 1]
```diff
@@ -39,7 +39,7 @@ depends: [
   "sha" {>= "1.12"}
   "cpu"
   "arg-complete"
-  "yaml"
+  "yaml" {>= "3.0.0"}
```
Comment (sim642) [inline packages/goblint/goblint.2.1.0/opam:42]: goblint.2.0.0 and goblint.2.0.1 should probably have the same lower bound.

**Comment by edwintorok** [comment]
I've reported the issue with Ruby on Fedora38 here: https://github.com/goblint/analyzer/issues/1158

File: packages/lintcstubs/lintcstubs.0.4.7/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
+]
+dev-repo: "git+https://github.com/edwintorok/lintcstubs.git"
+pin-depends: [
+	["dune-compiledb.0.4.0" "git+https://github.com/edwintorok/dune-compiledb.git#0.4.0"]
+]
```
Comment (sim642) [inline packages/lintcstubs/lintcstubs.0.4.7/opam:41]: AFAIK opam-repository doesn't allow `pin-depends`, so you'll have to release this package first.

**Comment by edwintorok** [comment]
Looks like some workers ran out of space now: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/e13fec6036ee48e189c605c13b41ed95e2c8fed8/variant/compilers,4.11,lintcstubs.0.4.7
`Sys_error("No space left on device")`

**Comment by avsm** [comment]
@edwintorok you can run `ocaml-ci-local` from https://github.com/ocurrent/ocaml-ci. This won't run some of the tests, but it's a good way to get most of the architectures (via qemu) for local testing.

**Comment by edwintorok** [comment]
The only failures now are on experimental distros because 'gcc' is not really gcc but 'clang', I think this is ready to be reviewed and merged now. Thanks for bearing with me while I was fixing the other CI errors.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**